.github/workflows/build_and_test.yaml

```diff
@@ -15,3 +15,4 @@ jobs:
       enable-common-libs: true
       #apt-cache-version: v0
       #py-cache-version: v0
+      python-version: "3.8"
```

Comment:
Just remembered one more thing, 

https://github.com/StackStorm-Exchange/stackstorm-git/blob/202441564a20f4b9c5d03f6cba3553fcfa2a9049/.github/workflows/release.yaml#L12-L20

needs adding `python-version: "3.8"` too

Reply:
Done. Thank you for all your Feedback :)